.github/workflows/edr-benchmark-feat-solidity-tests.yml

```diff
@@ -19,50 +19,63 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/feat/solidity-tests' }}
 
 jobs:
-  js-benchmark:
-    name: Run JS scenario runner benchmark
+  benchmarks-test:
+    name: Benchmarks test
     environment: github-action-benchmark
     runs-on: self-hosted
     # Only run for trusted collaborators since third-parties could run malicious code on the self-hosted benchmark runner.
-    if: github.ref == 'refs/heads/feat/solidity-tests' || (github.event.pull_request.author_association == 'OWNER' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'COLLABORATOR')
+    if: github.ref == 'refs/heads/main' || github.repository == github.event.pull_request.head.repo.full_name
```

Comment:
`main` looks like it might be changed by mistake

Reply:
Apologies, I got confused a bit. It actually needs to be `feat/solidity-tests` here to make it run in the branch, but we want to change it back to `main` before we merge `feat/solidity-tests` into main. Fixed it here with a TODO comment: [Fix precondition for soltests benchmark](https://github.com/NomicFoundation/edr/pull/524/commits/ae8625fa3457eb5d4bd1f402959881bc91c2015f)